src/cosalib/aws.py

```diff
@@ -111,6 +112,20 @@ def aws_run_ore_replicate(build, args):
 
 @retry(reraise=True, stop=stop_after_attempt(3))
 def aws_run_ore(build, args):
+     # Capture any input from image.yaml
+    image_yaml = flatten_image_yaml(
+        '/usr/lib/coreos-assembler/image-default.yaml',
+        flatten_image_yaml('src/config/image.yaml')
+    )
+    if 'aws-imdsv2-only' in image_yaml and image_yaml['aws-imdsv2-only'] :
+        ore_args.extend(['--imds-support'], "v2.0")
```

Comment:
The code in upload.go suggests this is a boolean argument:

```suggestion
        ore_args.extend(['--imdsv2-only'])
```

Reply:
Hmm, it is. I was under the impression that the ore_args.extend used the aws's api? so we had to be specific to what ever AWS needed?

Reply:
We're calling `ore` here so we need to use whatever format of option the upload.go code is requiring. IIUC:

```
cmdUpload.Flags().BoolVar(&uploadIMDSv2Only, "imdsv2-only", false, "enable IMDSv2-only support")
```

requires us to pass `--imdsv2-only`. Maybe I'm missing something.

`ore` then calls the underlying code that talks to the AWS API.

Reply:
No, I was confused. I see what you mean now. thank you.